### ISSUE TITLE
MSBuild host picked by user is not honoured

### ISSUE THREAD

**Issue by alfonsogarciacaro** [ionide/ionide-vscode-fsharp#508]
When opening an F# netcore2 project I see the following in msbuild output:

```
[11:35:59 INFO ] MSBuild (.NET) found at /Library/Frameworks/Mono.framework/Versions/Current/Commands/msbuild
[11:35:59 INFO ] Dotnet cli (.NET Core) found at /usr/local/share/dotnet/dotnet
[11:35:59 INFO ] MSBuild (.NET) activated
```

So I select `Pick MSBuild Host` in the F# Project Explorer and select the .NET Core one:

```
[12:12:35 DEBUG] user choose host .NET Core (/usr/local/share/dotnet/dotnet msbuild)
```

However, if I try to run a task from the F# Project Explorer like `Clean` I see the following:

```
[12:13:08 INFO ] invoking msbuild from /Library/Frameworks/Mono.framework/Versions/Current/Commands/msbuild on "/Users/alfonsogarciacaronunez/dev/Fable/src/templates/simple/Content/src/FableTemplate.fsproj" for target Clean
```

The .NET (Framework) MSBuild host is still being picked in spite of the user choice. This later fails with the following message:

```
/Library/Frameworks/Mono.framework/Versions/5.0.1/lib/mono/msbuild/15.0/bin/Sdks/Microsoft.NET.Sdk/build/Microsoft.NET.Sdk.DefaultItems.targets(165,5): error : Duplicate 'Compile' items were included. The .NET SDK includes 'Compile' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file. For more information, see https://aka.ms/sdkimplicititems. The duplicate items were: 'App.fs' 
```

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `2.34.0`

**Comment by alfonsogarciacaro** [comment]
Awesome, it's working! Thanks for the ultra-fast fix!
